components/home/Home.bs

```diff
@@ -20,11 +21,80 @@ sub init()
 
     m.homeRows = m.top.findNode("homeRows")
     m.homeRows.focusBitmapBlendColor = chainLookupReturn(m.global.session, "user.settings.colorCursor", ColorPalette.HIGHLIGHT)
+
+    ' Create random background task
+    m.randomBackgroundTask = createObject("roSGNode", "RandomBackgroundTask")
```

Comment:
We have a setting to set a background image. Wouldn't it make more sense to add a new "random item backdrop" option to that setting?

![WIN_20250703_15_48_17_Pro](https://github.com/user-attachments/assets/17f9c47d-c083-4adf-9c06-3245c4c634a4)